docs/notifications.md

```diff
@@ -0,0 +1,96 @@
+# Notifications
+
+### Notification Types
+
+#### Package Releases
+
+Private Packagist will notify you every time we find one or more new releases for one of your packages.
```

Comment:
Can you explain what "releases" are? It should really say "versions" here and explain that tags, branches, commits are new or modified versions?

Reply:
done